selfdrive/locationd/test/test_ublox_processing.py

```diff
@@ -0,0 +1,77 @@
+import unittest
+
+import numpy as np
+
+from laika import AstroDog
+from laika.helpers import UbloxGnssId
+from laika.raw_gnss import calc_pos_fix, correct_measurements, process_measurements, read_raw_ublox
+from tools.lib.logreader import LogReader
+from tools.lib.route import Route
+
+
+def get_gnss_measurements(log_reader):
+  gnss_measurements = []
+  for msg in log_reader:
+    if msg.which() == "ubloxGnss":
+      ublox_msg = msg.ubloxGnss
+      if ublox_msg.which == 'measurementReport':
+        report = ublox_msg.measurementReport
+        if len(report.measurements) > 0:
+          gnss_measurements += [read_raw_ublox(report)]
+  return gnss_measurements
+
+
+class TestUbloxProcessing(unittest.TestCase):
+
+  @classmethod
+  def setUpClass(cls):
+    path = Route("4cf7a6ad03080c90|2021-09-29--13-46-36").log_paths()[0]
```

Comment:
Does this work in CI? Other tests use `selfdrive.test.openpilotci.get_url` to build an URL to the CI bucket.